horizon/manifest.jsonnet

```diff
@@ -27,7 +27,7 @@ kpm.package({
 
     network: {
       external_ips: [],
-      port: 80,
+      port: 8080,
```

Comment:
Right, for some reason, with the Newton image:
```
+ apache2 -DFOREGROUND
(13)Permission denied: AH00072: make_sock: could not bind to address 0.0.0.0:80
no listening sockets available, shutting down
AH00015: Unable to open logs
```

Trying to figure out why now.